### PR TITLE
Move UASD to markdown

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

* converted the /legal/ubuntu-advantage-service-description page to use markdown

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/legal/ubuntu-advantage-service-description](http://0.0.0.0:8001/legal/ubuntu-advantage-service-description)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Compare to the existing live page, see that only the internal links are highlighted and that all copy is the same.

### CODE REVIEW

**Comment by pmahnke** [comment]
@cjohnston1158 	can you look as well?

**Comment by nottrobin** [comment]
@pmahnke could you fix the linting error?

```
$ sass-lint 'static/sass/**/*.scss' --verbose --no-exit -i static/sass/prism.scss

static/sass/_layout_legal_pages.scss
  1:1  error  Mixin 'layout_legal_pages' should be written in lowercase with hyphens  mixin-name-format

static/sass/_pattern_desktop-statistics.scss
  36:76   warning  Domains in URLs are disallowed                                               no-url-domains
  36:132  warning  Domains in URLs are disallowed                                               no-url-domains
  87:31   warning  Color functions such as 'rgba' should only be used in variable declarations  no-color-literals

static/sass/styles.scss
  66:1  error  Mixin 'layout_legal_pages' should be written in lowercase with hyphens  mixin-name-format

✖ 5 problems (2 errors, 3 warnings)
```

**Comment by pmahnke** [comment]
@nottrobin 	fixed

**Comment by nottrobin** [comment]
@pmahnke you still didn't fix the linting:

```
yarn run v1.9.4
warning package.json: License should be a valid SPDX license expression
$ sass-lint 'static/sass/**/*.scss' --verbose --no-exit -i static/sass/prism.scss

static/sass/_layout_legal_pages.scss
  1:1  error  Mixin 'layout_legal_pages' should be written in lowercase with hyphens  mixin-name-format

static/sass/_pattern_desktop-statistics.scss
  36:76   warning  Domains in URLs are disallowed                                               no-url-domains
  36:132  warning  Domains in URLs are disallowed                                               no-url-domains
  87:31   warning  Color functions such as 'rgba' should only be used in variable declarations  no-color-literals

static/sass/styles.scss
  66:1  error  Mixin 'layout_legal_pages' should be written in lowercase with hyphens  mixin-name-format
```

File: templates/legal/shared/_appendix-support-scope.md [hunk 1]
```diff
@@ -0,0 +1,439 @@
+<h1 id="appendix-support-scope">Appendix {{number}} - Support scope</h1>
```
Comment (nottrobin) [inline templates/legal/shared/_appendix-support-scope.md]: This has gone from this:

image

To this:

image

That feels like a regression. You may need to add some more explicit markup here to make the spacing better. Although it's a shame that Vanilla doesn't treat `<h1>` followed by `<h2>` better.

File: templates/legal/ubuntu-advantage-service-description/index.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    <h4>Severity Level 4 — Non-urgent request</h4>
+
+    Level 4 requests include cosmetic issues, informational requests, feature requests, and similar matters. Canonical does not provide a timeline or guarantee for inclusion of any feature requests. Canonical will review each level 4 case and determine whether it is a product enhancement to be considered for a future release, an issue to be fixed in the current release or an issue to be fixed in a future release. Canonical will review and respond to information requests with a reasonable level of effort during coverage hours. Canonical may close cases representing level 4 issues after responding if Canonical believes it is appropriate to do so.
+
+    <h3>Response times</h3>
+
+    - Canonical will use reasonable efforts to respond to support requests made by the customer within the response times set forth below, based on the applicable service and severity level.
+    - “Business hours” are defined as 08:00 - 18:00 Monday - Friday local to the customer’s headquarters unless another location is agreed. All times exclude public holidays.
+
+    Table of target initial response times
```
Comment (nottrobin) [inline templates/legal/ubuntu-advantage-service-description/index.md]: This used to be centred:

image

Now isn't:

image

I do think the latter looks objectively worse. May be worth adding some custom markup just for this item:


```suggestion
    <p style="text-align: center">Table of target initial response times</p>
```

**Comment by cjohnston1158** [comment]
I don't know if these are new but:
- [ ] Doesn't seem like there's a proper title on the page
- [ ] Still not a fan of how "3. Severity levels and target response times" wraps
- [ ] The spacing between lines with a number looks off. 
issue1
- [ ] Reference links don't seem to be working i.e. "24x7 self-service customer care portal and knowledge base (see Section 2)" - the "Section 2" link doesn't seem to be working.
- [ ] 1. Service initiation has two 1.1's

**Pull request merged**